### PR TITLE
Reformatted code,added new css/js file/modified queries and added function to determine status for each gallery instance

### PR DESCRIPTION
**Pull request by Chauwaj**
…instance in gallerQueries,added new css and js file

### CODE REVIEW

**Comment by Chauwaj** [comment]
@Jessonsotoventura  and @memo330179  so I had messed up my files and branch while trying to fix the merge conflicts before i could make a new pull request. My files where duplicated and showing up in wrong locations in the file tree.It took me hours to try and fix so I just created a new work space and committed all my work on a new branch in the interest of time.

File: app/controllers/admin/all_galleries.py [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
 
 @admin.route('/', methods=["GET","POST"])
 def all_galleries():
-    user = Forms.get(fid=1)
-    name = user.first_name
-    print(name)
-    print(doesUserHaveRole("admin"))
-    return render_template('views/admin/all_galleries.html')
+    
+    gallery_submission_querry = (Galleries
```
Comment (Jessonsotoventura) [inline app/controllers/admin/all_galleries.py]: This looks like a query that could go in either the GalleriesQuery as something like: get_all_galleries or in form queries as get_all_forms_group_by_galleries

File: app/models/queries/GalleryQueries.py [hunk 2]
```diff
@@ -67,4 +68,33 @@ def update(gid, title, open_date, close_date, description, banner):
         return gid
     except Exception as e:
         print (e)
-    return None
\ No newline at end of file
+    return None
+    
+def getstatus(gid):
```
Comment (Jessonsotoventura) [inline app/models/queries/GalleryQueries.py]: pip8 is based on underscores, two words should be separated using underscores, so get_status not getstatus or getStatus

**Review by Jessonsotoventura — APPROVED** [review]
Great work! 
There are a few minor things, but I will handle all those in the merge.

File: app/controllers/admin/all_galleries.py (hunk 1 above)
Comment (Chauwaj) [reply app/controllers/admin/all_galleries.py]: Thanks @Jessonsotoventura  Noted.

File: app/models/queries/GalleryQueries.py (hunk 2 above)
Comment (Chauwaj) [reply app/models/queries/GalleryQueries.py]: Noted

File: app/controllers/admin/all_galleries.py (hunk 1 above)
Comment (Jessonsotoventura) [reply app/controllers/admin/all_galleries.py]: No I think you were correct, after thinking more about where to move it, this query is a bit too specific to move into the GalleryQueries.

File: app/controllers/admin/all_galleries.py [hunk 3]
```diff
@@ -1,14 +1,30 @@
...
+                                .select(Galleries, fn.Count(Forms.fid).alias('count'))
+                                .join(Forms, JOIN.LEFT_OUTER)
+                                .group_by(Galleries))
+         
+    gallery_submissions_count ={}  
+    status= {}
+    
+    for gallery in gallery_submission_querry:
+        status[str(gallery)]= getstatus(gallery.gid)
```
Comment (Jessonsotoventura) [inline app/controllers/admin/all_galleries.py]: Please use gallery.title instead of str(gallery)

**Review by Jessonsotoventura — APPROVED** [review]

**Pull request merged**